### PR TITLE
🥊 Resilient testing + log gathering

### PR DESCRIPTION
**Pull request by major**
Convert the bash script to an ansible playbook so we can gracefully
handle testing failures and gather logs reliably. Colorful output
is nice, too.

Signed-off-by: Major Hayden <major@redhat.com>

### CODE REVIEW

**Comment by major** [comment]
schutzbut_fedora32_base

**Comment by major** [comment]
schutzbut_fedora32_base

**Comment by major** [comment]
Looks like we're running into the `libmodulemd` bug from https://bugzilla.redhat.com/show_bug.cgi?id=1822030.

**Comment by major** [comment]
schutzbut_fedora32_base

**Comment by major** [comment]
schutzbot_fedora32_base

**Comment by major** [comment]
schutzbot_fedora32_base

**Comment by major** [comment]
schutzbot_fedora32_base

File: jenkins/run_tests.sh [hunk 1]
```diff
@@ -1,44 +1,44 @@
 #!/bin/bash
 set -euxo pipefail
 
-# Ensure Ansible is installed.
-sudo dnf -y install ansible
+# Restart systemd to work around some Fedora issues in cloud images.
```
Comment (ondrejbudai) [inline jenkins/run_tests.sh:4]: Can you be more specific?

Comment (major) [reply jenkins/run_tests.sh:4]: Ah, I'll update that.

**Pull request merged**